tests/mozdef_util/test_elasticsearch_client.py

```diff
@@ -120,6 +120,54 @@ def test_search_no_results(self):
         assert results['hits'] == []
 
 
+class TestCloseIndex(ElasticsearchClientTest):
+
+    def teardown(self):
+        super(TestCloseIndex, self).teardown()
+        if pytest.config.option.delete_indexes:
+            self.es_client.delete_index('test_index')
+
+    def test_writing_to_closed_index(self):
```

Comment:
This function name is misleading, since we aren't "writing" anything.

Reply:
I don't think it's misleading, it's exactly what the test is attempting to do. 
Maybe adding what you feel is more appropriate would be more helpful. 
Do you want to see something like test_failure_writing_to_closed_index?

Reply:
Oh nvm! I see what happened :) good catch!